src/window.rs

```diff
@@ -118,7 +118,11 @@ where
         B: FnMut(&egui::Context, &mut Queue, &mut State),
         B: 'static + Send,
     {
-        let renderer = get_renderer(window);
+        let renderer = Renderer::new(window).unwrap_or_else(|err| {
+            // TODO: better error log and not panicking, but that's gonna require baseview changes
+            log::error!("oops! the gpu backend couldn't initialize! \n {err}");
+            panic!()
```

Comment:
You should panic with an error message too, in case the user hasn't set up a logger for their app.